projects/lib/src/oauth-service.ts

```diff
@@ -1216,7 +1216,7 @@ export class OAuthService extends AuthConfig {
         }
     }
 
-    private storeAccessTokenResponse(
+    public storeAccessTokenResponse(
```

Comment:
I feel like this should be a separate PR

Reply:
In ideal world it might be. But i have really lack of time in playing with git now tring to split it into 2 merge requests + it is not breaking change it is just open one essential method to public. Maybe we can merge it like it is this time? :)